ArchiSteamFarm/Storage/GlobalConfig.cs

```diff
@@ -398,8 +396,11 @@ public sealed class GlobalConfig {
 	[UsedImplicitly]
 	public bool ShouldSerializeSSteamOwnerID() => !Saving;
 
+	// TODO: Pending removal, Statistics property which got changed into RemoteConnection, we never serialize it after update
+#pragma warning disable CA1822 // We can't mark it as static
```

Comment:
Your comment does not state the "why?" 😕

Reply:
Abry pls

Reply:
You're nit-picky with my merge requests and I'm returning the favor 😉

Reply:
Be nit-picky with the code that is actually going to stay in ASF for indefinite future, not with one that will be gone in a month from now from the repo :wink: